modules/eks/karpenter/README.md

```diff
@@ -47,9 +42,17 @@ components:
         atomic: true
         wait: true
         rbac_enabled: true
+        chart_values:
+          aws:
+            enableENILimitedPodDensity: false
```

Comment:
According to the [discussion in Slack](https://kubernetes.slack.com/archives/C02SFFZSA2K/p1689893172133639?thread_ts=1689849266.459519&cid=C02SFFZSA2K) the plan is to remove this setting and always set it to `true` unless overridden by `maxPods`. Since the number of pods depends on the size of the machine, it's probably best to leave it auto configured by default.

Either change it to:
```suggestion
            enableENILimitedPodDensity: true
```

or remove the setting entirely.

Reply:
removed